client/websocket/websocket.go

```diff
@@ -331,7 +335,12 @@ func wsLoadCandles(s *Server, cl *wsClient, msg *msgjson.Message) *msgjson.Error
 	cl.feedMtx.RLock()
 	feed := cl.feed
 	cl.feedMtx.RUnlock()
-	if feed.host != req.Host || feed.base != req.Base || feed.quote != req.Quote {
+	// If market hasn't been initialized/chosen yet (client should do it in a separate
+	// 'loadmarket' request), or if client wants to change currently chosen market (requesting
+	// candles for market that's different from currently chosen implies that) - we can
+	// try to load it here.
+	if feed == nil ||
+		(feed.host != req.Host || feed.base != req.Base || feed.quote != req.Quote) {
 		var msgErr *msgjson.Error
 		feed, msgErr = loadMarket(s, cl, &req.marketLoad)
```

Comment:
One additional consideration, 

with the "lazy market loading/reloading logic" implemented in this PR do we even need to support (or issue from UI-client) `loadmarket` msg ?

Reply:
When I remove [this line](https://github.com/LasTshaMAN/dcrdex/blob/86db1f4e2a8ebb52c5041c3c16b7618ab742bd03/client/webserver/site/src/js/markets.ts#L795-L795), `/markets` page seems to be working same way without it,

that's because it loads candlesticks as well, and I thought maybe having markets page without candlesticks doesn't make much sense - hence maybe remove it; but maybe there is a use-case for a market without candlesticks (like showing order-book depth without candles)

Reply:
Oh that's an interesting point!  Before the latest markets page redesign, candlesticks weren't shown unless you toggled to them.  But now it does `this.loadCandles()` -> `this.requestCandles()` -> `ws.request('loadcandles',...` which would be seem to be redundant with the 'loadmarket' you linked to.  Will defer to @buck54321 on that, but I think you're correct.